LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan.NetworkServer/UdpServer.cs

```diff
@@ -1,413 +0,0 @@
-                    case PhysicalIdentifier.PullData:
-                        if (this.pullAckRemoteLoRaAggregatorPort == 0)
-                        {
-                            this.pullAckRemoteLoRaAggregatorPort = receivedResults.RemoteEndPoint.Port;
-                            this.pullAckRemoteLoRaAddress = receivedResults.RemoteEndPoint.Address.ToString();
-                        }
-
-                        SendAcknowledgementMessage(receivedResults, (int)PhysicalIdentifier.PullAck, receivedResults.RemoteEndPoint);
-                        break;
-
-                    // This is a PUSH_DATA (upstream message).
-                    case PhysicalIdentifier.PushData:
-                        SendAcknowledgementMessage(receivedResults, (int)PhysicalIdentifier.PushAck, receivedResults.RemoteEndPoint);
-                        DispatchMessages(receivedResults.Buffer, startTimeProcessing);
-
-                        break;
-
-                    // This is a ack to a transmission we did previously
-                    case PhysicalIdentifier.TxAck:
-                        if (receivedResults.Buffer.Length == 12)
-                        {
-                            Logger.Log(
-                                "UDP",
-                                $"packet with id {ConversionHelper.ByteArrayToString(receivedResults.Buffer.RangeSubset(1, 2))} successfully transmitted by the aggregator",
-                                LogLevel.Debug);
-                        }
-                        else
-                        {
-                            var logMsg = string.Format(
-                                    CultureInfo.InvariantCulture,
-                                    "packet with id {0} had a problem to be transmitted over the air :{1}",
-                                    receivedResults.Buffer.Length > 2 ? ConversionHelper.ByteArrayToString(receivedResults.Buffer.RangeSubset(1, 2)) : string.Empty,
-                                    receivedResults.Buffer.Length > 12 ? Encoding.UTF8.GetString(receivedResults.Buffer.RangeSubset(12, receivedResults.Buffer.Length - 12)) : string.Empty);
-                            Logger.Log("UDP", logMsg, LogLevel.Error);
-                        }
-
-                        break;
-                    case PhysicalIdentifier.PushAck:
-                    case PhysicalIdentifier.PullResp:
-                    case PhysicalIdentifier.PullAck:
-                    case PhysicalIdentifier.Unknown:
-                    default:
-                        Logger.Log("UDP", "unknown packet type or length being received", LogLevel.Error);
-                        break;
-                }
-            }
-        }
-
-        private void DispatchMessages(byte[] buffer, DateTime startTimeProcessing)
-        {
-            try
-            {
-                var messageRxpks = Rxpk.CreateRxpk(buffer);
```

Comment:
This method seems to be unused now, should we move it to the test code entirely/remove it?

Reply:
Imho we should rename/migrate the create RXPK and rxpk format with an associated cleanup. I will create a follow up work item for this

Reply:
Thanks, in that case I'll resolve this comment since we'll tackle it in another PR.